kustomize/overlay/7.1.0/securebanking/defaults/securebanking/kustomization.yaml

```diff
@@ -2,9 +2,9 @@ namespace: default
 commonLabels:
   app.kubernetes.io/name: "forgerock"
 resources:
-- ../kustomizeConfig
-- ../7.1.0/ingress
-- ../ig
+- https://github.com/ForgeRock/forgeops/kustomize/base/ingress?ref=release/7.1.0
+- https://github.com/ForgeRock/forgeops/kustomize/base/ig?ref=release/7.1.0
```

Comment:
Another PR to upgrade to 7.2.0 please!

Reply:
Sure, wanted to get 7.1.0 working as that's what we're currently on, once tested and okay'd can move to 7.2.0

Reply:
Sorry, I meant issue, not PR!